ElectronClient/app/gui/PromptDialog.jsx

```diff
@@ -155,6 +191,20 @@ class PromptDialog extends React.Component {
 				timeFormat={time.timeFormat()}
 				onChange={(momentObject) => onDateTimeChange(momentObject)}
 			/>
+		} else if (this.props.inputType === 'tags') {
+			inputComp = <CreatableSelect
+				styles={styles.tagList}
+				theme={styles.tagListTheme}
+				ref={input => this.answerInput_ = input}
```

Comment:
Please use React.createRef() to create references (the other methods are deprecated).